web/svelte.config.js

```diff
@@ -4,12 +4,6 @@ import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
   preprocess: vitePreprocess(),
-  onwarn: (warning, handler) => {
-    if (warning.code.includes('a11y')) {
-      return;
-    }
-    handler(warning);
-  },
```

Comment:
Is this change related to the async linting?

Reply:
Yes, indirectly. I updated SvelteKit to start type-checking the `svelte.config.js` file, which was necessary to get the new eslint rules to work. It didn't like the code here because the types of `warning` and `handler` were not defined. At first I was going to fix it by adding the types here, but then I noticed that when I removed it that I didn't see any warnings printed, so I felt like that was the better fix since it's sort of bad practice to hide the warnings rather than fix them

Reply:
Sounds good!